tests/responses_callbacks.py

```diff
@@ -21,7 +30,7 @@ def generic_callback(request):
     qs = parse_qs(req_body)
     qs = {k: v[0] for k, v in qs.items()}
 
-    if qs.get('payload-test', None) is not None:
+    if qs.get('payload-test') is not None:
```

Comment:
since you get it multiple times, just store it as a variable before the conditional